app/controllers/users/invitations_controller.rb

```diff
@@ -8,8 +8,9 @@ def invite_resource
   end
 
   def accept_resource
-    resource = resource_class.accept_invitation!(update_resource_params)
-    resource
+    # resource = resource_class.accept_invitation!(update_resource_params)
+    # resource
+    resource_class.accept_invitation!(update_resource_params)
```

Comment:
Just wondering whether we need to keep this commented out code?

Reply:
Commented out code has now been removed.